NewForce-Cohort-7/tabloidfullstack-numpty-buggers
# Js comment

Description
This pull request is for [Trello Ticket #20](https://trello.com/c/XaT3eRaP/20-20-create-a-comment)

The branch for the pull request: js-comment

Updates
Users can now create new comments on posts.

Type of Change
New feature - non breaking

Testing instructions
Follow these instructions to protect the branch you are working on and to test this pull request: https://gist.github.com/jordan-castelloe/96f76d5bd4cfbd2181084af674c2a1af#when-you-need-to-test-a-teammates-pr

Seed the Data
If you do not currently have the code for this project, clone the repository - (https://github.com/NewForce-Cohort-7/tabloidfullstack-numpty-buggers)

Open the TabloidFullStack.sln in visual studio and prepare your database.

In the github repository: https://github.com/NewForce-Cohort-7/tabloidfullstack-numpty-buggers

Open the SQL folder: (https://github.com/NewForce-Cohort-7/tabloidfullstack-numpty-buggers/tree/main/SQL)

there are two files - [01_Tabloid_Create_DB.sql](https://github.com/NewForce-Cohort-7/tabloidfullstack-numpty-buggers/blob/main/SQL/01_Tabloid_Create_DB.sql), [02_Tabloid_Seed_Data.sql](https://github.com/NewForce-Cohort-7/tabloidfullstack-numpty-buggers/blob/main/SQL/02_Tabloid_Seed_Data.sql)

In Visual Studio in the top navbar click View and select SQL Server Object Explorer. Open the server marked as "SQLEXPRESS" in the SQL Server Object Explorer on the left side of Visual Code. Once there, add a new SQL server by selecting the 'Add SQL Server' option at the top of the side bar. Copy the data from the seed data and execute by pressing the green Play button.

start the server by clicking the solid green play button in the middle top of your screen. A browser window will display with the Tabloid landing page.

Step by Step Instructions
In your purple Visual Studio run the app by clicking the green play arrow in the top center of your screen.
The Swagger API Testing page will display.
In your terminal CD into TabloidFullStack folder and navigate to your src folder.
When in the folder run npm start
This will open you local host and you will see the Tabloid login page.
Login as "[foo@bar.com](mailto:foo@bar.com)", you do not need a password.
You will see the Tabloid home page.
Click on Posts on the navbar

Choose a post and click details 
click the "add comment button" 
Add a comment to the post and click submit 
you should be  directed back to the list of comments 
click on the post and choose "view comments" to confirm that your post was added
